debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/SourceRecordAssert.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver.util;
+
+import io.debezium.data.VerifyRecord;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.fest.assertions.Assertions;
+
+public class SourceRecordAssert {
```

Comment:
Nice! We should look into consolidating the different approaches going forward, i.e. see whether existing tests could make use of this, too. But not too urgent.

Reply:
Filed (low-prio) https://issues.jboss.org/browse/DBZ-1095 for this.